docs-src/go/how-to-develop-a-worker-in-go.md

```diff
@@ -5,19 +5,6 @@ sidebar_label: Develop Worker
 description: Develop an instance of a Worker by calling worker.New(), available via the go.temporal.io/sdk/worker package.
 ---
 
-The [Worker Process](/concepts/what-is-a-worker-process) is where Workflow Functions and Activity Functions are executed.
-
-- Each [Worker Entity](/concepts/what-is-a-worker-entity) in the Worker Process must register the exact Workflow Types and Activity Types it may execute.
-- Each Worker Entity must also associate itself with exactly one [Task Queue](/concepts/what-is-a-task-queue).
```

Comment:
Why remove this? and why in this PR unless maybe I am missing something?

Reply:
I just ran yarn gen. This probably got pulled in from somewhere else. Let me check.